airgen/src/lib.rs

```diff
@@ -220,31 +220,7 @@ fn utility_functions(asm_file: AnalysisASMFile) -> BTreeMap<AbsoluteSymbolPath,
     asm_file
```

Comment:
What is a utility function? Maybe a comment or a more descriptive name (if that makes sense)?